bcache/get.go

```diff
@@ -51,8 +51,21 @@ func NewFS(mountPoint string) (FS, error) {
 	return FS{&fs}, nil
 }
 
-// Stats retrieves bcache runtime statistics for each bcache.
+// Stats is a wrapper around _Stats
+// It returns full available statistics
 func (fs FS) Stats() ([]*Stats, error) {
+	return fs._Stats(true)
+}
+
+// StatsWithoutPriority is a wrapper around _Stats.
+// It ignores priority_stats file, because it is expensive to read.
+func (fs FS) StatsWithoutPriority() ([]*Stats, error) {
+	return fs._Stats(false)
+}
+
+// _Stats retrieves bcache runtime statistics for each bcache.
+// priorityStats flag controls if we need to read priority_stats.
+func (fs FS) _Stats(priorityStats bool) ([]*Stats, error) {
```

Comment:
Go convention is to use mixed case instead of underscores for variable and function names, so this should be `stats` or `getStats` or something similar as long as it starts with a lower case letter.